assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -68,6 +40,40 @@
 		}
 	}
 
+	&__btn {
+		&.esprs-button.ant-btn {
+			&#{$filter-btn} {
+				border: 2px solid var(--ee-color-grey-10);
+				box-shadow: none;
+				margin: 0;
+				&:focus,
+				&:hover {
+					border: 1px solid var(--ee-color-primary-low-contrast);
+					box-shadow: 0 0 0 1px var(--ee-color-primary-low-contrast);
+				}
+			}
+		}
+		&__wrap {
+			align-items: center;
+			display: flex;
+			flex-flow: column nowrap;
+			padding: var(--ee-padding-tiny);
+			width: 90px;
+			#{$filter-btn} {
+				&#{$filter-btn} {
+					&.ee-active-filters {
```

Comment:
same here, this was used to increase specificity, more info: https://www.tobymackenzie.com/blog/2016/05/10/duplicate-selectors-increase-specificity/

***
there might be a cleaner way to do this, dunno ATM